frontend/admin/src/js/components/user/UserTable.tsx

```diff
@@ -156,10 +163,18 @@ export const UserTable: React.FC = () => {
   return (
     <div data-h2-margin="b(top-bottom, m)">
       <h2 id="user-table-heading" data-h2-visibility="b(invisible)">
-        All Users
+        {intl.formatMessage({
+          defaultMessage: "All Users",
+          description: "Title for the admin users table",
+        })}
       </h2>
       <TableHeader
-        onSearchChange={(val: string | undefined) => setSearchState(val)}
+        onSearchChange={(term: string | undefined, col: string | undefined) =>
```

Comment:
Could you add some logic to reset the paginator on search?  Right now if you're on page 14 and you search and get less than 14 pages of results the paginator gets into an inconsistent state

https://user-images.githubusercontent.com/8978655/177560834-72dcb99d-2398-49fe-b70b-db8a3557c92d.mp4

.

Reply:
Yikes, this would have been bad. Thanks!

Fixed with [reset current page on search](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3260/commits/35aa35755f7da77299da114cbc0376c89e349aeb)